### ISSUE TITLE
Add loading/saving functions in DDPG/VPG/PPO1

### ISSUE THREAD

**Issue by Sharad24** [SforAiDl/JigglypuffRL#20]
Since we're using pytorch, this is an added advantage as we can save policies really easily unlike TF

Some pseudocode to help
#### Saving
`hyperparams = {'param1':value1, 'param2':value2, 'weights': model.state_dict()}`
`torch.save(hyperparams, 'path.pt')`

#### Loading
`hyperparams = torch.load('path.pt')`
Then its just a dictionary update to self for which there are plenty ways. E.g. Iterating over the dict and using `setattr`

**Comment by Zeus3101** [comment]
I tried implementing it in #37 . Can you review the code?
